tests/WE2E/machine_suites/coverage.hera.intel.nco

```diff
@@ -11,6 +11,7 @@ grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
 grid_RRFS_CONUScompact_3km_ics_HRRR_lbcs_RAP_suite_HRRR
 MET_ensemble_verification_only_vx_time_lag
 pregen_grid_orog_sfc_climo
+smoke_dust_grid_RRFS_CONUS_25km_suite_RRFS_sas
```

Comment:
Do we want to add the 25km smoke and dust WE2E test to a separate suite, or should we replace the 3km smoke and dust WE2E test that is currently present in `coverage.gaeac6`?

Reply:
I had assumed that AQM people still want the 3km test run as part of the coverage tests, so I'll leave this up to them. The main reason for including a 25km case is so that people wanting to run a cheap smoke test can, it was not meant to replace the 3km test entirely.

Reply:
Thanks for the explanation, @mkavulich!  We can go ahead and keep both in.